xenit-alfresco/templates/acs/acs-config.yaml

```diff
@@ -48,6 +48,9 @@ data:
   {{- if not .Values.solr.enabled }}
   GLOBAL_index.subsystem.name: "noindex"
   {{- end }}
+  {{- if ((((.Values.acs).email).inbound).port) }}
```

Comment:
why not {{- if .Values.acs.email.inbound.port }} ?

Reply:
Without the braces, rendering fails if any of the nested levels are nil -which is the case if we leave the value empty by default. https://stackoverflow.com/a/68807258